osh/builtin.py

```diff
@@ -1213,6 +1215,25 @@ def __call__(self, arg_vec):
     return status
 
 
+BIND_SPEC = _Register('bind')
+
+
+class Bind(object):
+  def __init__(self, readline_mod):
+    self.readline_mod = readline_mod
+
+  def __call__(self, arg_vec):
+    if len(arg_vec.strs) == 1:
+      raise args.UsageError('bind keyseq:function-name')
+
+    status = 0
+    for i in xrange(1, len(arg_vec.strs)):
```

Comment:
Does bash process all positional arguments?

When I do `help bind`, it doesn't have the customary ... syntax for the last arg:

[keyseq:readline-function or readline-command]

I've never used `bind` honestly.  Can you maybe write some manual test cases in the comments?

Ideally tests should be automatic, but interactive readline stuff is hard to test of course... I maintain some manual test cases here, and those would be useful for `bind`

https://github.com/oilshell/oil/wiki/Testing-the-Interactive-Shell